collection-scripts/gather_sno

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NODES_PATH=${OUT:-"${BASE_COLLECTION_PATH}/nodes"}
+
+function get_system_data_off_sno {
+
+    #Create a path for each node
+    mkdir -p "$NODES_PATH"/"$1"
+    
+    local DEBUG_POD_NAME_PREFIX=""
+
+    #Start Debug pod force it to stay up until removed in "default" namespace
+    oc debug --to-namespace="default" node/"$1" -- chroot /host /bin/bash -c 'sleep 300' > /dev/null 2>&1 &
+
+    #Debug pod name start with the name of the node (after removing all "." for the node name)
+    DEBUG_POD_NAME_PREFIX=${1//./}
+
+    #Mimic a normal oc call, i.e pause between two successive calls to allow pod to register
+    sleep 2
```

Comment:
Can't you use a wait for a condition (vs a sleep)? 

https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait see `--for condition=available`

Reply:
You do this below; why is this sleep needed?

Reply:
If there was no sleep then the pod will not register on time, and the 'oc wait' command will resolve with an error.
This what will happen if we omit the "sleep 2" command:
`[rbaturov@rbaturov must-gather]$ source collection-scripts/gather_sno
INFO: Waiting for SNO system data collection to complete ...
error: arguments in resource/name form must have a single resource and name
Debug pod for node sno2.r207-sno2.r207.lab.eng.cert.redhat.com never activated
[1]+  Done                    get_system_data_off_sno "${NODE}"
INFO: SNO system data collection complete.
[rbaturov@rbaturov must-gather]$ 
`